docs/src/main/sphinx/security/file-system-access-control.rst

```diff
@@ -37,9 +38,23 @@ contents:
    access-control.name=file
    security.config-file=etc/rules.json
 
+If the config should be loaded via the rest endpoint ``http://trino-test/config`` and
+is wrapped into a JSON object and available via the ``data`` key ``etc/access-control.properties``
+should look like this:
+
+.. code-block:: text
+
+   access-control.name=file
+   security.config-file=http://trino-test/config
+   security.json-pointer=/data
+
 The config file is specified in JSON format. It contains rules that define which
 users have access to which resources. The rules are read from top to bottom and
-the first matching rule is applied. If no rule matches, access is denied.
+the first matching rule is applied. If no rule matches, access is denied. A JSON
+pointer (RFC 6901) can be specified using the ``security.json-pointer`` property
+to specify a nested object inside the JSON content containing the rules. Per default,
+the file is assumed to contain a single object defining the rules rendering
+the specification of ``security.json-pointer`` unnecessary in that case.
```

Comment:
If it is unnecessary, should we throw an exception if the config is set in file mode ?